packages/manager/modules/hub/src/components/tile/tile.html

```diff
@@ -0,0 +1,44 @@
+<div class="oui-tile hub-tile">
+    <span
+        class="oui-heading_5 oui-tile__title"
+        data-ng-bind="::$ctrl.title"
+    ></span>
+    <span class="oui-chip hub-chip" data-ng-bind="$ctrl.badgeText"></span>
+    <button
+        class="oui-button oui-button_link hub-button"
+        data-ng-if="::(!$ctrl.href && !$ctrl.state)"
+        data-ng-click="$ctrl.onClick()"
+    >
+        <span data-ng-bind="::$ctrl.buttonText"></span>
+        <span
+            class="oui-icon oui-icon-arrow-right_extra-thin hub-button__right-arrow"
+            aria-hidden="true"
+        ></span>
+    </button>
+    <a
+        class="oui-button oui-button_link hub-button"
+        data-ng-if="!!$ctrl.href"
+        data-ng-href="{{$ctrl.href}}"
+        data-ng-attr-target="{{::$ctrl.linkTarget}}"
+        data-ng-click="$ctrl.onClick()"
+    >
+        <span data-ng-bind="::$ctrl.buttonText"></span>
+        <span
+            class="oui-icon oui-icon-arrow-right_extra-thin hub-button__right-arrow"
+            aria-hidden="true"
+        ></span>
+    </a>
+    <a
+        class="oui-button oui-button_link hub-button"
+        data-ng-if="::!!$ctrl.state"
+        data-ui-sref="{{::$ctrl.getFullSref()}}"
+        data-ng-click="$ctrl.onClick()"
+    >
+        <span data-ng-bind="::$ctrl.buttonText"></span>
+        <span
+            class="oui-icon oui-icon-arrow-right_extra-thin hub-button__right-arrow"
+            aria-hidden="true"
+        ></span>
+    </a>
```

Comment:
We don't seem to need all this, maybe we could just keep it as simple as possible and just pass a `href` and a `text` for this link

Reply:
This has been removed and we now have just a link with href and text